src/core/lib/channel/channelz.cc

```diff
@@ -90,28 +90,27 @@ void CallCountingHelper::RecordCallSucceeded() {
       static_cast<gpr_atm>(1));
 }
 
-CallCountingHelper::CounterData CallCountingHelper::Collect() {
-  CounterData out;
-  memset(&out, 0, sizeof(out));
+void CallCountingHelper::CollectData(CounterData* out) {
+  memset(out, 0, sizeof(*out));
```

Comment:
No need for this memset, since the fields of the struct are all initialized to 0.

Reply:
Done